buffer/src/test/java/io/netty/buffer/AlignedPooledByteBufAllocatorTest.java

```diff
@@ -63,4 +66,62 @@ public void testCorrectElementSize() {
         assertTrue(a.release());
         assertTrue(b.release());
     }
+    @Test
+    public void testDirectSubpageReleaseLock() throws Exception {
```

Comment:
Could we change our visibility of things to package-private so we dont need reflection for the test ?

Reply:
Yes,  this can be done by changing the field `PoolArena.smallSubpagePools` to package-private.

Reply:
Done.